src/gui/editors/PianoRoll.cpp

```diff
@@ -3357,7 +3357,7 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 			const int key_num = m_markedSemiTones.at(x);
 			const int y = keyAreaBottom() - 1 - m_keyLineHeight *
 				(key_num - m_startKey + 1);
-			if(y > keyAreaBottom()) { break; }
+			if(y >= keyAreaBottom() - 1) { break; }
```

Comment:
Not sure when the space got deleted here, but it should be there.
```suggestion
			if (y >= keyAreaBottom() - 1) { break; }
```

Reply:
Yep seems so.